packages/terra-dropdown-button/src/SplitButton.jsx

```diff
@@ -96,13 +137,22 @@ class SplitButton extends React.Component {
   }
 
   handleCaretKeyDown(event) {
-    if (event.keyCode === KeyCode.KEY_SPACE) {
+    if (event.keyCode === KeyCode.KEY_SPACE || event.keyCode === KeyCode.KEY_RETURN) {
       this.setState({ caretIsActive: true });
+      this.setState({ openedViaKeyboard: true });
+    } else if (event.keyCode === KeyCode.KEY_DOWN && this.state.isOpen && !this.state.openedViaKeyboard) {
```

Comment:
Please leave code comments mentioning that the focus behaviour differs when accessed via click v/s keypress.

Reply:
I have already [comments](https://github.com/cerner/terra-core/blob/3d4cbdca1a5c96ae211cdcf5bdb5847ecfdc965c/packages/terra-dropdown-button/src/SplitButton.jsx#L150) added for this do you want me change the text in it..?